helm/prometheus-meta-operator/templates/recording-rules/service-levels.rules.yml

```diff
@@ -10,41 +10,52 @@ spec:
   groups:
   - name: service-level
     rules:
-    - expr: "count(up{app='kubernetes'}) by (cluster_id)"
+    - expr: "count(up{app='kubernetes'}) by (cluster_type,cluster_id)"
       labels:
+        class: HIGH
         service: api-server
       record: raw_slo_requests
-    - expr: "sum((up{app='kubernetes'} * -1) + 1) by (cluster_id)"
+    - expr: "sum((up{app='kubernetes'} * -1) + 1) by (cluster_type,cluster_id)"
```

Comment:
Why do we group by `cluster_type` also now ?

Reply:
See https://github.com/giantswarm/g8s-prometheus/pull/1150

Reply:
Ok